api/v1/service/resource/resource.go

```diff
@@ -508,7 +508,7 @@ func versionInfoFromVersion(v model.ResourceVersion) *resource.ResourceVersion {
 func (s *service) GetRawYamlByCatalogKindNameVersion(ctx context.Context, p *resource.GetRawYamlByCatalogKindNameVersionPayload) (io.ReadCloser, error) {
 	s.Logger(ctx).Info(fmt.Sprintf("Fetching YAML for resource %s", p.Name))
 
-	yamlPath := fmt.Sprintf("%s/%s/%s/%s/%s", s.CatalogClonePath(), strings.ToLower(p.Catalog), strings.ToLower(p.Kind), p.Name, p.Version)
+	yamlPath := fmt.Sprintf("%s/%s/%s/%s/%s", s.CatalogClonePath(), p.Catalog, strings.ToLower(p.Kind), p.Name, p.Version)
```

Comment:
also can we try to maintain consistency? I can see that in one of the changes done above, we have removed the lower case conversion for `p.Kind` and here we have kept that